angular/core/components/thread_page/context_panel/context_panel.haml

```diff
@@ -52,4 +52,5 @@
     .context-panel__attachment{ng-repeat: "attachment in discussion.attachments() | orderBy:'createdAt' track by attachment.id"}
       %attachment_preview{attachment: "attachment", mode: "context"}
 
-  %a.context-panel__edit-link.lmo-card-minor-action.lmo-no-print{href: "", ng-click: "editThread()", translate: "thread_context.edit_thread", ng-if: "canEditThread()"}
+  %button.context-panel__add-comment.lmo-no-print{ng-click: "scrollToCommentForm()", translate: "thread_context.add_comment"}
```

Comment:
This needs to appear only if you can comment, yeah? (a la `/development/view_open_group_as_visitor`)